cli/valet.php

```diff
@@ -93,6 +100,28 @@
         info('Your Valet TLD has been updated to ['.$tld.'].');
     }, ['domain'])->descriptions('Get or set the TLD used for Valet sites.');
 
+    /**
+     * Get or set the loopback address currently being used by Valet.
+     */
+    $app->command('loopback [loopback]', function ($loopback = null) {
+        if ($loopback === null) {
+            return output(Configuration::read()['loopback']);
+        }
+
+        $oldLoopback = Configuration::read()['loopback'];
+
+        Configuration::updateKey('loopback', $loopback = trim($loopback, '.'));
+
+        DnsMasq::refreshConfiguration();
+        Site::resecureForNewLoopback($oldLoopback, $loopback);
+        Nginx::installServer();
+        PhpFpm::restart();
+        Nginx::restart();
+
+        info('Your valet loopback address has been updated to ['.$loopback.']');
+
+    }, ['address'])->descriptions('Get or set the loopback address used for Valet sites');
```

Comment:
Why is `['address']` here? This is the "alias" parameter. Putting it here means `valet loopback` and `valet address` become synonyms. Was that the intention? Why?

Reply:
Wrong refactoring, will be removed.